src/components/JournalPage/index.jsx

```diff
@@ -16,14 +16,25 @@ import PDFViewer from '../PDFViewer';
 
 class JournalPage extends React.Component {
   componentDidMount() {
+    this.trackVisit();
     this.props.getPage(this.props.match.params.pageId);
   }
   componentDidUpdate(prevProps) {
+    if (prevProps.userId !== this.props.userId) {
+      this.trackVisit();
+    }
     if (prevProps.match.params.pageId !== this.props.match.params.pageId) {
+      this.trackVisit();
```

Comment:
would we only want to track after we've confirmed the getPage() succeeded?

Reply:
i.e. on `fetchPageSuccess`

Reply:
That's a good point. Will fix.